docs/intro.txt

```diff
@@ -24,9 +27,6 @@ and Windows. To install, run:
 .. note::   `view_calendar` requires the extra-install-argument above. The method `get_calendar` can be used with the base installation of pytket-quantinuum.
 
 
-**Web Interface:**
-https://um.qapi.quantinuum.com/
-
```

Comment:
You could potentially keep this, as for pytket-quantinuum users this will be the only UI? That said I think the UMUI will be sunsetted in 2025